Documentation & Methods Download Page/docdownload.Rmd

```diff
@@ -0,0 +1,56 @@
+---
+title: "Measure & Methodology Documentation Download"
+output: 
+  html_document:
+    theme: cosmo
+    toc: true
+    toc_depth: 4
+    toc_float: true
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
```

Comment:
Also, could you add image optimization for both this page and the about page? The resulting about page file is far larger than it needs to be: https://stackoverflow.com/questions/38815784/knitr-html-output-too-large

Reply:
Yes - both files have been decreased in size drastically 🥲

Reply:
Turns out they reduced in size because they got rid of all the styling and moved those out to separate files of equivalent size -- looks like it's a necessary evil. I added some image optimization, but they're still pretty big, unfortunately.